server/routes/testutils/appSetup.ts

```diff
@@ -38,6 +38,7 @@ function appSetup(services: Services, production: boolean, userSupplier: () => H
     res.locals = {
       user: { ...req.user } as HmppsUser,
     }
+    req.body = req.body || {}
```

Comment:
Can be `undefined` in express 5.

Reply:
This stops it defaulting to `undefined` ?

Reply:
#561 removes it and all tests are perfectly happy still!